aurora/grids_utils.py

```diff
@@ -401,6 +403,127 @@ def create_time_grid_new(timing, verbose=False, plot=False):
     return time, save
 
 
+def ELM_time_grid(timing, ELM_model):
```

Comment:
@odstrcilt indeed, maybe this is too detailed to include in Aurora and should just stay in ImpRad, but someone like @antonellozito wants to use Aurora without entering the world of OMFIT, so he needed something.  A few options:
- @antonellozito keeps these new ELM-related functions for himself, without releasing them publicly (if so, Aurora remains with no explicit ELM-related functionality);
- you include the ELM-related functions from ImpRad into Aurora, so that @antonellozito can use those (this may make sense from the perspective of "ImpRad offers a GUI and integrated workflow; Aurora has the basic toolbox");
- we keep some degree of redundancy, with @antonellozito's ELM-related functions in Aurora and your functions in ImpRad.

Reply:
@odstrcilt as discussed privately with @fsciortino, for me it might be ok to adjust things in such a way to have my functions for simulating ELM just for myself, and include instead in Aurora what you did for IMPRAD. However, I just thought that, generally, it's not a bad idea to include in Aurora something, even very simple, to phenomenologically reproduce ELMs. Why should Aurora contain a sawteeth model, as it is in the current master version, but not an ELM model? :)
Also, as said before, I think that Aurora should be independent from the content of other OMFIT packages.